interface/super/rules/controllers/browse/view/plans_config.php

```diff
@@ -24,8 +24,8 @@
  * @link http://librehealth.io
 */
 
- require_once("../../../../../globals");
- require_once("$srcdir/headers.inc.php");
+require_once(dirname(__FILE__)."/../../../../../globals.php");
+require_once($GLOBALS['srcdir']."/headers.inc.php");
 ?>
```

Comment:
Same here. Just try that and get back to the PR to see what it gives

Reply:
@muarachmann I've tried changing that line to 
 require_once("$srcdir/headers.inc.php");
but it seems to give an error.